front-end/public/index.html

```diff
@@ -10,9 +10,7 @@
       content="Web site created using create-react-app"
     />
     <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
-    
-    
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+
```

Comment:
Thanks for that I saw the warning yesterday and I completely forgot to remove it